### PR TITLE
Apply to Keycloak documentation style to the Getting Started guides

### LINKED ISSUE #17133: Apply documentation standards to Getting Started Guides
### Description

Suggest changes to content to match the standards used by the other guides.

### Discussion

_No response_

### Motivation

The documentation should follow the same standards to provide a consistent look and feel.

### Details

We have the documentation standards documented here: https://github.com/keycloak/keycloak-documentation/blob/main/internal_resources/styleguide.adoc

### PR DESCRIPTION
**Pull request by andymunro**
Closes #17133

@stianst, @ahus1, @pedroigor, @DGuhr Please check these updates to conform the Getting Started guides to the Keycloak style guide: https://github.com/keycloak/keycloak-documentation/blob/main/internal_resources/styleguide.adoc

These changes make these guides match the other guides more closely. If you prefer, we could meet first to discuss some changes as part of a decision-driven documentation effort.

### CODE REVIEW

File: docs/guides/src/main/getting-started/getting-started-openshift.adoc [hunk 1]
```diff
@@ -9,43 +9,43 @@ summary="Get started with Keycloak on OpenShift">
...
++
 [source,bash,subs="attributes+"]
 ----
 oc login -u developer -p developer
 ----
 
-For this guide, we are going to create a new project called `keycloak`. For that, execute the following command:
+. Create a project called `keycloak` by entering the following command:
 
 [source,bash,subs="attributes+"]
```
Comment (ahus1) [inline docs/guides/src/main/getting-started/getting-started-openshift.adoc:40]: The bash script should relate to the enumeration item, therefore add a "+" to keep them together.

```suggestion
. Create a project called `keycloak` by entering the following command:
+
[source,bash,subs="attributes+"]
```

File: docs/guides/src/main/getting-started/getting-started-openshift.adoc [hunk 2]
```diff
@@ -55,35 +55,35 @@ oc process -f https://raw.githubusercontent.com/keycloak/keycloak-quickstarts/la
...
 ----
 NAME                READY     STATUS      RESTARTS   AGE
 keycloak-1-deploy   0/1       Completed   0          1h
 keycloak-1-l9kdx    1/1       Running     0          1h
 ----
 
-Once the server is provisioned, run the following command to find out the URLs of Keycloak:
+. Once the server is provisioned, enter the following command to find out the Keycloak URLs:
 
 [source,bash,subs="attributes+"]
```
Comment (ahus1) [inline docs/guides/src/main/getting-started/getting-started-openshift.adoc:88]: The bash script should relate to the enumeration item, therefore add a "+" to keep them together.

```suggestion
. Once the server is provisioned, enter the following command to find out the Keycloak URLs:
+
[source,bash,subs="attributes+"]
```

File: docs/guides/src/main/getting-started/templates/first-app.adoc [hunk 3]
```diff
@@ -1,36 +1,39 @@
...
-After the client is created you need to update the following values for the client:
+After the client is created, make these updates to the client:
 
-. Valid redirect URIs: `https://www.keycloak.org/app/*`
-. Web origins: `https://www.keycloak.org`
-
-Remember to click `Save`.
+. Scroll down to *Access settings*.
+* Set *Valid redirect URIs* to `https://www.keycloak.org/app/*`
+* Set *Web origins* to `https://www.keycloak.org`
```
Comment (ahus1) [inline docs/guides/src/main/getting-started/templates/first-app.adoc:21]: Those links are data that should be entered, and not clicked on. Therefore I added a passthrough so they are rendered as text and not as links. This was this way in the original docs, still it should be changed.

```suggestion
* Set *Valid redirect URIs* to `+https://www.keycloak.org/app/*+`
* Set *Web origins* to `+https://www.keycloak.org+`
```

**Review by ahus1 — CHANGES_REQUESTED** [review]
Hi @andymunro - thank you for bringing this guide into shape!

See below for minor formatting issues. I also reviewed all the wording changes and they look valid from my Keycloak knowledge.

**Comment by andymunro** [comment]
Thanks @ahus1 for the fast response. I have made these corrections.

**Review by ahus1 — APPROVED** [review]
Thanks for these edits, approving it.

**Comment by ahus1** [comment]
@stianst - as you suggested, I had a look how the updated guides will be rendered on the upstream website, and it looks good. 

image

**Review by stianst — APPROVED** [review]

**Pull request merged**